after/ftplugin/tex.vim

```diff
@@ -1,27 +1,32 @@
-let s:path = fnamemodify(resolve(expand('<sfile>:p')), ':h')
+silent !which evince
+redraw!
```

Comment:
Thanks! Could perhaps these two lines be replaced by `:help executable()`?

Reply:
Great idea! Added a commit.